packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.kt

```diff
@@ -305,6 +305,66 @@ public open class ReactViewGroup public constructor(context: Context?) :
       return false
     }
 
+    // For accessibility services (TalkBack), check if hover is within any child's hitSlop area.
+    // Only apply this logic when accessibility services are enabled to avoid interfering with
+    // other input methods (VR, mouse, stylus, etc.)
+    val accessibilityManager = context.getSystemService(Context.ACCESSIBILITY_SERVICE) as? AccessibilityManager
+    if (accessibilityManager?.isTouchExplorationEnabled == true &&
```

Comment:
This is still pretty expensive to do on a per-input basis (I think it needs to a Binder call)

Reply:
Ah yeah that makes sense. I've added some internal state to `ReactViewGroup` and a [TouchExplorationStateChangeListener](https://developer.android.com/reference/android/view/accessibility/AccessibilityManager.TouchExplorationStateChangeListener) to manage it.